async-nats/src/jetstream/stream.rs

```diff
@@ -321,6 +341,35 @@ pub struct StreamState {
     pub consumer_count: usize,
 }
 
+/// A raw stream message in the representation it is stored.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct RawMessage {
+    /// Subject of the message.
+    #[serde(rename = "subject")]
+    pub subject: String,
+
+    /// Sequence of the message.
+    #[serde(rename = "seq")]
+    pub sequence: u64,
+
+    /// Data of the mssage.
+    #[serde(default, rename = "data")]
+    pub data: String,
```

Comment:
why not bytes?

Reply:
Yeah base64 encoded string, with terrible documentation from legacy nats.rs (from me.. gotta admit) 😁

Reply:
ah, ok.

Yes, `TryInto` is a nice pattern from those fallible transofmartions we have in library.